src/libhirte/common/cfg.c

```diff
@@ -338,3 +339,55 @@ const char *cfg_dump(struct config *config) {
         }
         return cfg_info;
 }
+
+int cfg_agent_def_conf(struct config *config) {
+        char *buff = NULL;
+
+        // NodeName
+        if (cfg_set_value(config, CFG_NODE_NAME, get_hostname()) != 0) {
+                return -1;
+        }
+
+        // ManagerHost
+        if (cfg_set_value(config, CFG_MANAGER_HOST, HIRTE_DEFAULT_HOST) != 0) {
+                return -1;
+        }
+
+        // HeartbeatInterval
+        if (asprintf(&buff, "%li", AGENT_HEARTBEAT_INTERVAL_MSEC) <= 0) {
+                return -1;
+        }
+
+        if (cfg_set_value(config, CFG_HEARTBEAT_INTERVAL, buff) != 0) {
+                return -1;
+        }
+
+        // ManagerPort
+        if (asprintf(&buff, "%i", HIRTE_DEFAULT_PORT) <= 0) {
+                return -1;
+        }
+        if (cfg_set_value(config, CFG_MANAGER_PORT, buff) != 0) {
+                return -1;
+        }
+
+        return 0;
+}
+
+int cfg_manager_def_conf(struct config *config) {
+        char *buff = NULL;
+
+        // ManagerPort
+        if (asprintf(&buff, "%i", HIRTE_DEFAULT_PORT) <= 0) {
```

Comment:
Let's also convert `HIRTE_DEFAULT_PORT` into string so we don't need always to convert it